diff --git a/package.json b/package.json
index 1e516cc60cc55281aeb218832011ad337c323540..a191e614ba6d3d1d38144fcc97301e5d44ab007f 100644
--- a/package.json
+++ b/package.json
@@ -2,5 +2,6 @@
   "name": "deps-1",
   "private": true,
   "dependencies": {
+    "left-pad": "1.3.0"
   }
 }
diff --git a/src/a.ts b/src/a.ts
index 3cb72cdb1645221cc58dd5c81ffd11f826740e3b..6e0b435f66fc51ba8bd1fe140cee69cc66877aef 100644
--- a/src/a.ts
+++ b/src/a.ts
@@ -1 +1,3 @@
-export const a = (): string => 'a'
+import leftPad from 'left-pad'
+
+export const a = (): string => leftPad('a', 3)
