diff --git a/src/a.py b/src/a.py
index 4693ad3cf8b0903b98497fb89b8b524fbf1b93f4..92a07a9421c98dbb9422369c7d3a790201024d61 100644
--- a/src/a.py
+++ b/src/a.py
@@ -1,2 +1,6 @@
 def add(a, b):
     return a + b
+
+
+def double(a):
+    return a + a
diff --git a/tests/test_a.py b/tests/test_a.py
index ee5c2485ed15e78906b1e40ced5aafc20d243dd2..b5cd48d31a9bec3e50cf7c051b7923647cd35747 100644
--- a/tests/test_a.py
+++ b/tests/test_a.py
@@ -3,7 +3,4 @@ from src.a import add
 
 def test_one():
     assert add(1, 2) == 3
-
-
-def test_two():
     assert add(0, 0) == 0
